config/main.py

```diff
@@ -1015,6 +1015,32 @@ def load_minigraph(no_service_restart):
     click.echo("Please note setting loaded from minigraph will be lost after system reboot. To preserve setting, run `config save`.")
 
 
+#
+# config monitor_tx THRESHOLD 40
+# config monitor_tx POOLING_PERIOD 20
```

Comment:
Remove extra # and move those lines to be near @config.command("tx_error_monitor")